src/libs/Permissions.js

```diff
@@ -14,7 +14,7 @@ Onyx.connect({
  * @returns {Boolean}
  */
 function canUseAllBetas() {
-    return _.contains(betas, CONST.BETAS.ALL);
+    return true || _.contains(betas, CONST.BETAS.ALL);
```

Comment:
I will remove these before finalizing the PR.

Reply:
Oh, I am going to create a PR so that all betas are enabled in development, so we don't have to do this anymore